common/GPodderEpisodeListModel.qml

```diff
@@ -120,10 +114,10 @@ ListModel {
             query = currentCustomQuery;
         }
 
-        ready = false;
+        myself.ready = false;
```

Comment:
`this.ready = false;`
would seem to do the job too or am I wrong?

Reply:
you're right about that, but the other call 3 lines below needs it, because the context of the callback changed.
The problem here is, that I can't use `episodeListModel` anymore because that id is set externally, but I still need to access the top-level object of this file...